.github/workflows/docs/requirements.txt

```diff
@@ -1,3 +1,3 @@
 sphinx ~= 4.3.2
-sphinx_book_theme ~= 0.3.3
+sphinx_book_theme ~= 1.0.1
```

Comment:
Ah I see. I just want to make sure we get at least 1.0.1 here as there was a bug with 1.0.0 I want to avoid

https://github.com/executablebooks/sphinx-book-theme/issues/711

Will ~= 1.0 still give me the right version?

Reply:
Makes sense, pushed a change for that